### ISSUE TITLE
How to draw an accuracy curve after train?

### ISSUE THREAD

**Issue by kongzewuxin** [WuJie1010/Facial-Expression-Recognition.Pytorch#14]
After training the prgram,I want to draw an accuracy curve,but I don't to do.can you tell me the way.
And,how to use SVM to obeserve the training results.Thanks very much!!!

**Comment by WuJie1010** [comment]
I have not save the accuracy data, you should save the accuracy and draw. 
I only use a linearSVM to classify;
try “criterion = nn.MultiMarginLoss”

**Comment by kongzewuxin** [comment]
Hi，wujie：
 In the recent experiment, it was found by drawing the curve of loss and accuracy that in the FER2013data set, loss first dropped and then rose on the public test set and the private test set，but the acc kept rising until it remained unchanged. Was it caused by overfitting?
